search/lib/cmr/cmr_converter.js

```diff
@@ -1,6 +1,6 @@
 const _ = require('lodash');
-const { wfs, generateAppUrl, extractParam, generateSelfUrl } = require('./util');
-const cmr = require('./cmr');
+const { wfs, generateAppUrl, extractParam, generateSelfUrl } = require('../util');
+const cmr = require('./cmr.js');
```

Comment:
you shouldn't need the `.js` extension on this require statement.

Reply:
I made these changes.